### PR TITLE
Flu 16: QR Code for payment 

### PR DESCRIPTION
**Pull request by Gift-Stack**
Gotta merge #7 before this : )

### CODE REVIEW

File: app/app/send-modal.tsx [hunk 1]
```diff
@@ -68,6 +79,16 @@ export default function SendModal() {
     setIsLoadingTransfer(false);
 
     router.back();
+
+    // Refetch transactions
+    const res = await getPayments(user!.token, {
```
Comment (limone-eth) [inline app/app/send-modal.tsx]: I don't remember here why we need the list of transactions. Can you remind me?

**Review by limone-eth — APPROVED** [review]

File: app/app/send-modal.tsx (hunk 1 above)
Comment (Gift-Stack) [reply app/app/send-modal.tsx]: After making a transaction, we'd need to update the list of the state of the payments to include that transaction. 

It's a background API call, and wouldn't disrupt any other action.

Comment (limone-eth) [reply app/app/send-modal.tsx]: Can't you just do a pop and push in place?

Comment (Gift-Stack) [reply app/app/send-modal.tsx]: yeah, just did that ! Thanks for pointing that out

-- I didn't realize the implementation behind the transfer page list display

**Pull request merged**